api/assessment/assessment.proto

```diff
@@ -31,10 +31,10 @@ package clouditor;
 import "google/api/annotations.proto";
 import "google/protobuf/empty.proto";
 import "google/protobuf/timestamp.proto";
-import "evidence.proto";
-import "metric.proto";
+import "api/evidence/evidence.proto";
+import "api/assessment/metric.proto";
 
-option go_package = "api/assessment";
+option go_package = "clouditor.io/clouditor/api/assessment;assessment";
```

Comment:
Is the ";assessment" at the end intended?

Reply:
That is the syntax for <fullpath>;<relative_path>. This way the imports are resolved correctly according to the full path and the files are placed in the package according to the relative_path.

Reply:
Nice to know. Do you have a link to DOCs pointing that out?

Reply:
Looks like it might not be needed any more. https://developers.google.com/protocol-buffers/docs/gotutorial#defining-your-protocol-format. Seems the last part of the path will be used by default.

Reply:
I removed them